packages/app/src/cli/models/app/loader.ts

```diff
@@ -905,17 +905,12 @@ async function loadAppConfigurationFromState<
           ...configState.basicConfiguration,
         }
         delete file.path
-        const appVersionedSchema = getAppVersionedSchema(specifications)
+        const appVersionedSchema = getAppVersionedSchema(specifications, true)
```

Comment:
Should `allowDynamicallySpecifiedConfigs` be the default for this function? 🤔 
I see we call `getAppVersionedSchema` in three other places and it defaults to false